ics-frontend/src/app/app.component.html

```diff
@@ -1,4 +1,6 @@
+<div [class.alternative]="otherTheme">
```

Comment:
You don't need a separate div here, you can define the conditional class directly on `app-container`

Reply:
adjusted